sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/ddl.scala

```diff
@@ -20,15 +20,23 @@ package org.apache.spark.sql.execution.datasources
 import org.apache.spark.sql._
 import org.apache.spark.sql.catalyst.TableIdentifier
 import org.apache.spark.sql.catalyst.catalog.{CatalogTable, CatalogUtils}
-import org.apache.spark.sql.catalyst.plans.QueryPlan
-import org.apache.spark.sql.catalyst.plans.logical.{Command, LogicalPlan}
+import org.apache.spark.sql.catalyst.expressions.Attribute
+import org.apache.spark.sql.catalyst.plans.logical.LogicalPlan
 import org.apache.spark.sql.execution.command.{DDLUtils, RunnableCommand}
 import org.apache.spark.sql.types._
 
+/**
+ * Create a table and optionally insert some data into it. Note that this plan is unresolved and
+ * has to be replaced by the concrete implementations during analyse.
```

Comment:
To the other reviewers, `CreateTable` is a unified unresolved logical representation of CREATE TABLE (AS SELECT). It will be replaced by `CreateHiveTableAsSelectCommand`, `CreateTableCommand`, `CreateDataSourceTableCommand` or `CreateDataSourceTableAsSelectCommand`.